fennb/phirehose/.gitignore

```diff
@@ -0,0 +1,4 @@
+.project
+*.un~
```

Comment:
these files should be in the vendor

Reply:
yeah  the original version had a problem so i had to fix the bug .... . so i had to remove it from the vendor file 



if you use composer to install it when running the code it would give you an error